mcs/class/corlib/Makefile

```diff
@@ -49,7 +49,7 @@ REFERENCE_SOURCES_FLAGS += -d:FEATURE_MACL
 endif
 
 ifndef AOT_FRIENDLY_PROFILE
-REFERENCE_SOURCES_FLAGS += -d:FEATURE_REMOTING,MONO_COM,FEATURE_COMINTEROP,FEATURE_ROLE_BASED_SECURITY
+REFERENCE_SOURCES_FLAGS += -d:FEATURE_REMOTING,MONO_COM,FEATURE_COMINTEROP,FEATURE_ROLE_BASED_SECURITY,FEATURE_CONSOLE
```

Comment:
This is not REFERENCE_SOURCES flag, secondly make it profile filter specific as reusing AOT_FRIENDLY_PROFILE seems pretty wrong